geopandas/plotting.py

```diff
@@ -188,34 +189,17 @@ def plot_point_collection(ax, geoms, values=None, color=None,
     x = geoms.x.values
     y = geoms.y.values
 
+    # scatterplot uses different way to specify size as default plot markersize
+    if markersize:
+        markersize = markersize ** 2
+
```

Comment:
My idea was to keep the kwarg 'markersize' in our API. As this is what was used before, and otherwise would break code. Of course we could deprecate 'markersize' and say to users to use 's' from now on. But personally I find 'markersize' a much more clear name than 's' (certainly because you are as a user not directly using scatter)

But I agree that the squaring can be a bit surprising. But then I saw the matplotlib scatter docs that also say they use markersize**2 as the default (https://matplotlib.org/api/pyplot_api.html#matplotlib.pyplot.scatter), and then thought this would be OK :-) 
But if we keep it, should certainly add a note to the docstring